cli_tools/gce_onestep_image_import/main.go

```diff
@@ -44,6 +49,10 @@ func main() {
 	}
 }
 
+func printHelpLink() {
+	log.Println("Reference to https://cloud.google.com/compute/docs/import/import-aws-image for more detailed help.")
```

Comment:
"Refer to..." maybe sounds a bit more natural?

Reply:
done